src/components/__tests__/Home.spec.js

```diff
@@ -97,18 +55,18 @@ describe("Home", () => {
     );
     expect(rows[3].findAll("td")[1].text()).toBe("wikidata_qid");
 
-    // click on Skip to Next button
-    await wrapper.findAll("button")[1].trigger("click");
+    //   // click on Skip to Next button
+    //   await wrapper.findAll("button")[1].trigger("click");
 
-    // Expect the table to be updated with the next task details
-    const updatedRows = wrapper.find("tbody").findAll("tr");
-    expect(rows[0].findAll("td")[1].text()).toBe("toolforge-authors");
-    expect(rows[1].findAll("td")[1].text()).toBe(
-      "This project is aimed at getting very basic author statistics for a specified page."
-    );
-    expect(rows[2].findAll("td")[1].text()).toBe(
-      "https://toolsadmin.wikimedia.org/tools/id/authors"
-    );
-    expect(rows[3].findAll("td")[1].text()).toBe("openhub_id");
+    //   // Expect the table to be updated with the next task details
```

Comment:
We shouldn't merge commented out code. Do we want to dispose off the entire test or can we fix it?

Reply:
The current test will at least test that, when given certain props, the table will render correctly.  So it's not entirely without value, but a test for HomeView would be more useful.

Reply:
Addressed with [b3895cc](https://github.com/wikimedia/toolhunt-ui/pull/41/commits/b3895ccc14df0fc0d4bc2bec6257fa73d9fa4ecb)